framework/src/st_things/things_stack/framework/things_data_manager.h

```diff
@@ -102,7 +102,7 @@ const char *dm_get_svrdb_file_path(void);
 const char *dm_get_certificate_file_path(void);
 const char *dm_get_privatekey_file_path(void);
 const char *dm_get_filename(int filenum);
-const char *dm_get_things_device_type(int device_id);
+const char *dm_get_things_device_type(const char* device_id);
```

Comment:
for cording style,
`const char *device_id`

Reply:
ok